types/foundry/client/global.d.mts

```diff
@@ -14,7 +13,6 @@ declare global {
         export import CONST = constants;
         export import Hooks = foundry.helpers.Hooks;
         export import PIXI = PixiJS;
-        export import TinyMCE = tinymce;
```

Comment:
I think we can keep this global, right?

Reply:
Yeah probably. I wasn't sure if importing was preferred with the new file structure.

Reply:
so global or alias it as "tinymce"

Reply:
global works best I think.